README.md

```diff
@@ -17,7 +18,7 @@ Ember Custom Actions is compatible with Ember 2.8 (and higher) applications..
 
 ### Configuration
 
-You can define your custom default options in your `config/environment.js` file.
+You can define your custom options in your `config/environment.js` file
```

Comment:
I suppose this is more a philosophical question, but Isn't it that this add-on defines **default** options (inside `addon/config.js`), and by doing this `module.exports` you're in fact overriding them with your **custom** ones?